packages/core/schema/lavamoat-policy.v0-0-1.schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "LavaMoat Policy Schema",
+  "$id": "lavamoat-policy.v0-0-1",
+  "type": "object",
+  "description": "Schema for LavaMoat policy files",
+  "properties": {
+    "resources": {
+      "description": "Describe the resources available to your application and direct dependencies",
+      "title": "Resources",
+      "type": "object",
+      "$comment": "This one or more valid npm package names delimited by a \">\" character",
+      "patternProperties": {
+        "^(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*(>(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*)*$": {
+          "$ref": "#/$defs/resourcePolicy"
+        }
+      },
+      "additionalProperties": false
+    },
+    "resolutions": {
+      "type": "object",
+      "title": "Resolutions",
+      "description": "Custom run-time module resolutions by direct dependency",
+      "patternProperties": {
+        "^(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*(>(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*)*$": {
+          "type": "object",
+          "patternProperties": {
+            "^(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*(>(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*)*$": {
+              "type": "string",
+              "pattern": "^(\\.{1,2})(/(?=[^/\\0])[^/\\0]+)*/?$",
+              "minLength": 1,
+              "description": "Relative POSIX path to module from package root"
+            }
+          },
+          "additionalProperties": false
+        }
+      },
+      "additionalProperties": false
+    }
+  },
+  "additionalProperties": false,
+  "anyOf": [{ "required": ["resources"] }, { "required": ["resolutions"] }],
```

Comment:
I don't recall seeing resolutions used. @kumavis do we need them for anything?

Reply:
[This](https://github.com/LavaMoat/LavaMoat/pull/74/commits/b2045184c7edc523b5e459704676bd95ff21d8cb) is where resolutions were added.  They don't seem to interact with yarn's `resolutions` or npm's `overrides` in any way.